lib/ui/screen/root_drawer.dart

```diff
@@ -33,13 +36,13 @@ class RootDrawer extends ConsumerWidget {
         final nextTab = RootTab.values[value];
         switch (nextTab) {
           case RootTab.home:
-            const HomeRoute().go(context);
+            navigationShell.goBranch(HomeBranch.index);
           case RootTab.sessions:
-            const SessionsRoute().go(context);
+            navigationShell.goBranch(SessionsBranch.index);
           case RootTab.sponsors:
             const SponsorsRoute().go(context);
```

Comment:
I think it would be better to transition with `goBranch`.

Reply:
Thanks, adopted. It makes these process simpler.
[feat: define branchIndex in RootTab and use it in navigation rail and…](https://github.com/FlutterKaigi/conference-app-2023/pull/142/commits/273120cd4f556cfc1e845c0e5a2bac10ea07f96d)